src/components/Dropzone/FileListComponent.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+    <div class="files-layout">
+      <FileItemComponentVue
+        v-for="file, index in fileList"
+        :key="index"
+        :fileItem="file"
```

Comment:
vì cái component chung, chỗ prop FileItem có chắc chắn là cái object File {} khi upload từ máy tính không?. nếu có bài toán là render list file từ API thì cái API nó không có format như cái object File {} từ máy tính thì sao???